app/decorators/controllers/decidim/pages_controller_decorator.rb

```diff
@@ -0,0 +1,14 @@
+Decidim::PagesController.class_eval do
+  alias_method :original_index, :index
+
+  def index
+    if current_organization.host == "reptes.dca.cat"
+      @topic = Decidim::StaticPageTopic.find_by("title->>'ca' = 'La DCA'")
+      @page = current_organization.static_pages.first
+      enforce_permission_to :read, :public_page, page: @page
+      @pages = @topic&.pages
```

Comment:
if we add a `render partial: "who_we_are"` don't we avoid having to override index and create an `_original_index.html.erb`?
```suggestion
      @pages = @topic&.pages
      render partial: "who_we_are"
```